frame/benchmarking/src/lib.rs

```diff
@@ -1792,6 +1807,16 @@ macro_rules! add_benchmark {
 							.expect("benchmark name is always a valid string!")
 					);
 					None
+				},
+				Err($crate::BenchmarkError::Weightless) => {
+					$crate::log::error!(
+						"WARNING: benchmark weightless skipped - {}",
+						$crate::str::from_utf8(benchmark)
+							.expect("benchmark name is always a valid string!")
+					);
+					let mut dummy = $crate::BenchmarkResult::default();
+					dummy.components = selected_components.clone();
+					Some(vec![dummy.clone(), dummy.clone()])
```

Comment:
Why do we need to insert this twice?

Reply:
I thought the linear regression logic needs at least two values; but it also works with just one. Changed it.